source/_integrations/fan.xiaomi_miio.markdown

```diff
@@ -490,3 +490,11 @@ Turn the dry mode off.
 | Service data attribute    | Optional | Description                                             |
 |---------------------------|----------|---------------------------------------------------------|
 | `entity_id`               |       no | Only act on a specific Xiaomi miIO fan entity.          |
+
+## Troubleshooting `Unable to find device` error messages
+
+Check if the device is in the same subnet as the Home Assistant instance. Otherwise, you should configure your router/firewall to put this device in the same VLAN as the Home Assistant instance.
+
+If it's not possible to use VLANs for some reason, your last resort may be using NAT translation, between the IPs.
```

Comment:
From rule 498 until 500 isn't something for the docs, but fits better on the community forum. 

The main reason for this is that if we explain troubleshooting in OPNsense, we will soon have to do the same for all other router brands. Doesn't make it easier to maintain this. So can you remove it and post on the forum?

Reply:
I understand you point. Considering this documentation only currently supports for the yaml implementation, the troubleshooting I see as part of the learning curve. Specially, with the increasing complexity of our home networks.

The issue is not about the opnsense of pfsense support, just a limitation of this kind of hardware that may not be visible to the end user, until it connects its perfectly fine device in a fine tuned network and it simply does not work.

I see this kind of troubleshooting to be a part of the current documentation, avoiding recurrent new threads on the forums or empty ones, as this kind of issue is quite difficult to diagnose, if not this knowledge be shared promptly along the config of this integration. In one spot, end-to-end.

Reply:
If the text is not entirely accepted, I'm willing to correct it accordingly.

Reply:
The initial paragraph is fine. The others are exact solutions, which are out of scope for our documentation.

We don't want specific instructions for each router or firewall ending up here.